docs/core/deploying/native-aot/diagnostics.md

```diff
@@ -10,7 +10,10 @@ ms.date: 08/07/2023
 
 Native AOT shares some, but not all, diagnostics and instrumentation capabilities with CoreCLR. Because of CoreCLR's rich selection of diagnostic utilities, it's sometimes appropriate to diagnose and debug problems in CoreCLR. Apps that are [trim-compatible](../trimming/prepare-libraries-for-trimming.md) shouldn't have behavioral differences, so investigations often apply to both runtimes. Nonetheless, some information can only be gathered after publishing, so Native AOT also provides post-publish diagnostic tooling.
 
-## .NET 8 Native AOT diagnostic support
+## Native AOT diagnostic support
+
+> [!NOTE]
+> Starting from .NET 9, Native AOT supports targeting iOS-like platforms with .NET MAUI for which diagnostics support is described in the following [document](https://learn.microsoft.com/dotnet/maui/deployment/nativeaot?view=net-maui-9.0&preserve-view=true#native-aot-diagnostic-support-on-ios-and-mac-catalyst).
```

Comment:
I think it would be more consistent with our documentation style to add this link in "See also" section at the end of the doc. Random example from the existing docs https://learn.microsoft.com/en-us/dotnet/core/deploying/trimming/trim-self-contained#see-also.

Majority of this doc should be applicable to iOS too. The only wrinkle that I can see is that this doc talks explicitly about CoreCLR in several places. You can fix those by saying e.g. "full .NET runtime" so that it applies to Mono as well.

Reply:
Also, left comment on your other PR at https://github.com/dotnet/docs-maui/pull/2614#discussion_r1838437503

Reply:
Fixed.